src/Acquisition/tests.py

```diff
@@ -1620,25 +1655,31 @@ def test_cant_persist_acquisition_wrappers_classic():
     >>> pickler = cPickle.Pickler(file, 1)
 
     >>> def persistent_id(obj):
+    ...     if not hasattr(obj, '_p_oid'): return None
```

Comment:
Is this due to a difference between the pure-Python and C implementations of `Persistent`?